gprotorch/data_featuriser/featurisation.py

```diff
@@ -0,0 +1,122 @@
+import numpy as np
+from rdkit.Chem import MolFromSmiles, AllChem, Descriptors
+import pandas as pd
+from rxnfp.transformer_fingerprints import (
+    get_default_model_and_tokenizer,
+    RXNBERTFingerprintGenerator,
+)
+from drfp import DrfpEncoder
+from sklearn.feature_extraction.text import CountVectorizer
+import selfies as sf
+import graphein.molecule as gm
+from rdkit.Chem import rdMolDescriptors
+
+
+# Reactions
+def one_hot(df):
```

Comment:
I guess we should probably use this Args style format for docstrings since we've used it elsewhere: https://github.com/leojklarner/gauche/blob/main/gprotorch/kernels/gnn_kernels/pretrained_kernel.py